src/pages/youthboard/village-camp-survyey/[villageSurveyName].tsx

```
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from 'react';
+import { TENANT_DATA } from '../../../../app.config';
+
+
+const villageSurveyName = () => {
+    const { t } = useTranslation();
+    const router = useRouter();
+    const theme = useTheme<any>();
+    const { villageSurveyName } = router.query;
+    const [value, setValue] = React.useState(1);
+    const [village, setVillage] = useState<string>('');
+    const [camp, setCamp] = useState<string>('');
```

ESLint

[error] 'camp' is assigned a value but never used.

(no-unused-vars)